portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,27 +61,37 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String msg = getParameter(request, "comment-input", "");
 
     if (student) {
-      populateArrayList(comments, name, "student", msg);
+      populateDataStore(datastore, name, "student", msg);
     } else if (industry) {
-      populateArrayList(comments, name, "industry professional", msg);
+      populateDataStore(datastore, name, "industry professional", msg);
     } else if (recruiter) {
-      populateArrayList(comments, name, "recruiter", msg);
+      populateDataStore(datastore, name, "recruiter", msg);
     } else {
-      populateArrayList(comments, name, "other", msg);
+      populateDataStore(datastore, name, "other", msg);
     }
 
     //redirect back to original page
     response.sendRedirect("/index.html#comment");
   }
 
-  /** Helper function to add the name, category, and message to the right ArrayList. */
+  /** Helper function to add the name, category, and message to the ArrayList. */
   private void populateArrayList(List c, String name, String type, String msg) {
```

Comment:
if this method is not being called anymore, you can delete it

Reply:
fixed!